virtio-bindings/build.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright Red Hat, Inc.
+// SPDX-License-Identifier: (BSD-3-Clause OR Apache-2.0)
+
+use std::env;
+use std::path::PathBuf;
+
+fn generate_bindings(header: &str) {
```

Comment:
I think generating the bindings on crate build is imposing quite a heavy requirement on the host as bindgen has clang dependencies that are not easily resolvable. We probably didn't see it here since the rust-vmm container has a working combination of clang toolchain and bindgen tool but we're seeing that when we try to compile using GH actions using the "regular" rust action script.